drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/ScenarioRunnerServiceTest.java

```diff
@@ -17,45 +17,83 @@
 package org.drools.workbench.screens.testscenario.backend.server;
 
 import java.lang.annotation.Annotation;
+import java.net.URL;
 import java.util.ArrayList;
+
 import javax.enterprise.event.Event;
 
+import org.drools.compiler.kie.builder.impl.KieContainerImpl;
+import org.drools.core.ClockType;
+import org.drools.core.SessionConfiguration;
 import org.drools.workbench.models.testscenarios.shared.Scenario;
 import org.drools.workbench.screens.testscenario.model.TestScenarioResult;
 import org.guvnor.common.services.shared.test.TestResultMessage;
 import org.guvnor.structure.server.config.ConfigurationService;
-import org.jboss.errai.security.shared.api.identity.User;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.kie.api.KieServices;
+import org.kie.api.builder.ReleaseId;
 import org.kie.api.runtime.KieSession;
 import org.kie.workbench.common.services.backend.session.SessionService;
 import org.kie.workbench.common.services.shared.project.KieProject;
 import org.kie.workbench.common.services.shared.project.KieProjectService;
 import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
 import org.mockito.Mock;
+import org.mockito.Spy;
 import org.mockito.runners.MockitoJUnitRunner;
 import org.uberfire.backend.vfs.Path;
-
-import static org.junit.Assert.*;
-import static org.mockito.Mockito.*;
+import org.uberfire.backend.vfs.PathFactory;
+import org.uberfire.io.IOService;
+import org.uberfire.io.impl.IOServiceDotFileImpl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 @RunWith(MockitoJUnitRunner.class)
 public class ScenarioRunnerServiceTest {
 
+    private static final String MORTGAGES_HOME = "https://github.com/kiegroup/kie-wb-playground/tree/master/mortgages";
```

Comment:
@manstis @Rikkola how could/should I keep the up to date location here? Any ideas? I mean, the url now contains branch master, what could cause problems if this PR is cherry picked to other branch.

Reply:
@jomarko A JAR is created for https://github.com/kiegroup/kie-wb-playground which forms the examples that can be imported into the Workbench. It is versioned along with all other KIE artifacts. The Workbench includes the JAR as a dependency and unzips it to _load_ an example repository. You could consider doing the same for _mortgages_.